src/main/resources/static/index.html

```diff
@@ -148,7 +148,7 @@ <h1>Welcome to the Garden Linux Vulnerability Database</h1>
             <tr>
                 <td>
                     <form action="/getCveForDistribution" method="get" class="distribution">
-                        <label for="gardenlinuxVersion">...CVEs in any Garden Linux image</label>
+                        <label for="gardenlinuxVersion">...CVEs for a Garden Linux version</label>
```

Comment:
what I wanted to communicate here is that this also has an image filter, but it does filter on packages that are in any image. not sure if your text is more clear than mine.